styles/main.css

```diff
@@ -0,0 +1,199 @@
+body {
+  font-family: sans-serif;
+  width: 98%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 50px;
+}
+
+img {
+  margin: 10px;
+}
+
+body section {
+  border-bottom: solid 1px grey;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+#why {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+#page-intro h1 {
+  text-align: center;
+  font-size: 90px;
+  margin-top: 150px;
+  font-family: 'Bungee Outline', sans-serif;
+  color: purple;
+}
+
+#page-intro .button {
+  margin-bottom: 100px;
+}
+
+h2 {
+  text-align: center;
+  font-size: 28px;
+  margin-top: 40px;
+  margin-bottom: 20px;
+}
+
+ul {
+  list-style-type: none;
+  border: 0px;
+  padding: 0px;
+}
+
+sub {
+  font-size: 30px;
+  text-transform: capitalize;
+  font-weight: bold;
+  display: block;
+  text-align: center;
+  margin-top: 55px;
+  margin-bottom: 80px;
+}
+
+.button {
+  border: solid 4px;
+  border-radius: 12px;
+  border-color: lightgrey;
+  display: block;
+  width: 270px;
+  padding: 22px 20px 22px 20px;
+  font-size: 28px;
+  text-align: center;
+  text-decoration: none;
+  color: inherit;
+  margin: auto;
+}
+
+.button:hover {
+  background-color: grey;
+  color: white;
+}
+
+#intros {
+  border-bottom: none;
+}
+
+#intros img {
+  border-radius: 100%;
+  filter: grayscale(100%);
+  border: lightgray 1px solid;
+  padding: 5px;
+  display: block;
+  margin-left: 75px;
+  margin-right: 75px;
+}
+
+#intros, .quote {
+  padding-left: 8%;
+  padding-right: 8%;
+}
+
+#intros .name, .title {
+    display: block;
+    text-align: center;
+    margin: 15px;
+}
+
+#intros ul {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-top: 60px;
+}
+
+nav {
+  padding-left: 100px;
+  padding-right: 100px;
+}
+
+.nav-list {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr)
+}
+
+.nav-list li {
+  text-align: right;
+  padding: 10px;
+  font-weight: bold;
+  font-size: 20px;
+}
+
+#nav-title {
+  grid-area: 1 / span 3;
+  text-align: left;
+}
+
+.quote {
+  line-height: 2em;
+  font-family: serif;
+  font-style: italic;
+  font-size: 29px;
+  padding-top: 125px;
+  padding-bottom: 125px;
+  background-color: lightgrey;
```

Comment:
I like how the background color here breaks the page up visually as you move downwards.